config/env.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * This file has been ejected from create-react-app v3.4.0.
+ * It has not been touched since then (if you edit this file please remove this message).
```

Comment:
```suggestion
 * It has not been touched since then (if you edit this file please amend this message).
```

Reply:
Never mind... I'm guessing that wording is part of the eject.

Reply:
No I added that so I will amend these comments